src/Serilog/Enrichers/EnvironmentUserNameEnricher.cs

```diff
@@ -21,7 +21,11 @@
 namespace Serilog.Enrichers
 {
     /// <summary>
+#if USERNAMEFROMENV
```

Comment:
That seems to compile without any changes in the enricher now. I've removed that change.